cynic/src/integrations/chrono.rs

```diff
@@ -23,6 +23,8 @@ impl Scalar for NaiveDate {
     }
 }
 
+crate::impl_serializable_argument_for_scalar!(NaiveDate);
```

Comment:
Ah yes. Sorry - I forgot about this myself, otherwise I would have said in the original PR.

Unfortunate that we need it - did have a blanket `impl SerializableArgument for Scalar` at one point, but ran into some problems with it so switched to this.